bindings/python/tox.ini

```diff
@@ -48,3 +48,17 @@ deps =
 commands =
     pip install .
     pylint --disable=missing-function-docstring,missing-class-docstring --rcfile=.pylintrc tests/**/*.py 
+
+[testenv:all]
```

Comment:
I had the same thought. So that's what I tried at first. It's possible but not directly supported. You would need to call `tox -e {...}` multiple times which is more costly than this. The tipping point for me was that I wanted the formatter to modify in place, and not just show the diff.

Reply:
Alright, thanks